File: src/main/java/br/com/jiratorio/domain/vo/changelog/JiraChangelogItem.java
package br.com.jiratorio.domain.vo.changelog;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import java.time.LocalDateTime;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class JiraChangelogItem {

    private String field;

    private String fieldtype;

    private String from;

    private String fromString;

    private String to;

    private String toString;

    @JsonIgnore
    private LocalDateTime created;

}

File: src/main/java/br/com/jiratorio/domain/changelog/JiraChangelogItem.kt
package br.com.jiratorio.domain.changelog

import com.fasterxml.jackson.annotation.JsonIgnore
import java.time.LocalDateTime

data class JiraChangelogItem(
    var field: String? = null,

    var fieldtype: String? = null,

    var from: String? = null,

    var fromString: String? = null,

    var to: String? = null,

    var toString: String? = null,

    @JsonIgnore
    var created: LocalDateTime? = null
)
